ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -2669,3 +2669,21 @@ def validate_serviceexport():
     assert mon_count == len(
         get_mon_pods()
     ), f"Mon serviceexport count mismatch {mon_count} != {len(get_mon_pods())}"
+
+
+def check_storage_client_status(namespace=constants.OPENSHIFT_STORAGE_CLIENT_NAMESPACE):
```

Comment:
Storageclient is a different resource from Storagecluster. There should be created a new module for storageclients (ocs_ci/ocs/resources/storageclient.py).